install/README.md

```diff
@@ -213,6 +215,43 @@ A list of _datalogger_ sampling configurations for a given _station_ and recordi
 | _Start_ | Stream start time|
 | _Stop_ | Stream stop time|
 
+#### _GAINS_ ####
+ 
+A list of installation times where gains need to be applied to correct for local conditions, datalogger or sensor settings.
```

Comment:
Again, I find this a bit confusing.
If I understand correctly, `gains.csv` is going to be used to provide info around site specific corrections, whereas anything that is specific to the "recorder" (datalogger/sensor) will be represented in the calibration table.